engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1090,8 +1091,8 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                         break;
                     }
 
-                } catch (final OperationTimedoutException e) {
-                    s_logger.debug("Unable to send the start command to host " + dest.getHost());
+                } catch (OperationTimedoutException e) {
+                    s_logger.debug("Unable to send the start command to host " + dest.getHost()+" failed to start VM: "+vm.getUuid());
```

Comment:
Yes, but we have to many loglines which are debug and which should be error. Right now you have to run CloudStack on DEBUG to get any interesting information out of the logs. The VM deployment failed and that's an error.